### ISSUE TITLE
Only fixes one bridge

### ISSUE THREAD

**Issue by jidanni** [openstreetmap/iD#8873]
### URL

https://www.openstreetmap.org/edit#map=19/24.21053/120.88315

### How to reproduce the issue?

All I know is I crossed rivers twice,
but the helper system only offered to add one bridge.

### Screenshot(s) or anything else?

![20220104T003749](https://user-images.githubusercontent.com/1230959/147955974-54d0df2d-19fd-481d-8681-79adf22a36c1.jpg)

### Which iD Editor versions do you see the issue on?

Released version at openstreetmap.org/edit

### Which browsers are you seeing this problem on?

Chrome

**Comment by jidanni** [comment]
After it fixes the first spot, we just see:
![20220104T004155](https://user-images.githubusercontent.com/1230959/147956325-c951fb44-687b-4f4e-ad51-353df36dcac5.jpg)
And the first spot is still "throbbing."

**Comment by jidanni** [comment]
OK, I now, just for fun, clicked the second spot, and ah ha!
![20220104T004422](https://user-images.githubusercontent.com/1230959/147956598-c24f2aa6-3d5b-4372-ab2a-71811c6831b3.jpg)
Apparently when it inserted the first bridge, it chopped the road, so then it couldn't find the second spot.
OK, now I can go ahead and have it insert the second bridge for me. Good.

**Comment by matkoniecz** [comment]
<img width="836" height="318" alt="Image" src="https://github.com/user-attachments/assets/378f11aa-ca8c-4236-9f51-5df859966186" />
if you hover over warning it shows related river

**Comment by matkoniecz** [comment]
and mass changing all of cases makes no sense - road may cross 25 different streams and rivers

and automagic selecting one of ways is going to be confusing

I think it is fine as is

**Comment by tyrasd** [comment]
True, we want to have the user to address issues on an individual basis, so this is working as expected.
